time_weighted_average_price/on_chain/inputs/twap.json

```diff
@@ -1,27 +1,24 @@
 {
   "enclave_attested_application_public_key": {
-    "enclave_attestation": {
-      "Platform": "plain",
-      "PlAttests": [
-        "eyJEYXRhIjoiZXlKamRYSjJaVjkwZVhCbElqb2ljREkxTm1zeElpd2laR0YwWVNJNklrST0iLCJNZWFzdXJlbWVudCI6eyJQbGF0Zm9ybSI6InBsYWluIiwiQ29kZSI6InBsYWluIn19",
-        "eyJEYXRhIjoiUm5SSVIzRmllV2RrZW1seFNWaDRjbTB5YTFWNGNIWjFUWE4wU0RkUFdUUT0iLCJNZWFzdXJlbWVudCI6eyJQbGF0Zm9ybSI6InBsYWluIiwiQ29kZSI6InBsYWluIn19",
-        "eyJEYXRhIjoiY2xobFpYWmpjVEZsU2swM1MzTm5PSE5OYldKQlJrUlZURWR2Y0RKME1WZz0iLCJNZWFzdXJlbWVudCI6eyJQbGF0Zm9ybSI6InBsYWluIiwiQ29kZSI6InBsYWluIn19",
-        "eyJEYXRhIjoiVEhONGJub3pZMHRyVlZBeVJYcDBTVEZVTlU1Qlp6MGlmUT09IiwiTWVhc3VyZW1lbnQiOnsiUGxhdGZvcm0iOiJwbGFpbiIsIkNvZGUiOiJwbGFpbiJ9fQ==",
-        "eyJEYXRhIjoiWGpOV1o4ejJzMlZRZEl5b2R5NVNkVXEvajh6S0dKTmFUS3Rxb3MwaGdQdz0iLCJNZWFzdXJlbWVudCI6eyJQbGF0Zm9ybSI6InBsYWluIiwiQ29kZSI6InBsYWluIn19"
-      ]
-    },
-    "public_key": {
-      "curve_type": "p256k1",
-      "data": "BFtHGqbygdziqIXxrm2kUxpvuMstH7OY4rXeevcq1eJM7Ksg8sMmbAFDULGop2t1XLsxnz3cKkUP2EztI1T5NAg="
+    "enclave_attestation": "eyJQbGF0Zm9ybSI6InBsYWluIiwiUGxBdHRlc3RzIjpbImV5SkVZWFJoSWpvaVpYbEthbVJZU2pKYVZqa3daVmhDYkVscWIybGpSRWt4VG0xemVFbHBkMmxhUjBZd1dWTkpOa2xyU1QwaUxDSk5aV0Z6ZFhKbGJXVnVkQ0k2ZXlKUWJHRjBabTl5YlNJNkluQnNZV2x1SWl3aVEyOWtaU0k2SW5Cc1lXbHVJbjE5IiwiZXlKRVlYUmhJam9pVW01T1ExbFhaSEpNTURWTVQwVmtlRXN5ZUhGTmVrVXdXVEpLTUdSWVRrSk5SbWhOVmpCR1ZrMHdTVDBpTENKTlpXRnpkWEpsYldWdWRDSTZleUpRYkdGMFptOXliU0k2SW5Cc1lXbHVJaXdpUTI5a1pTSTZJbkJzWVdsdUluMTkiLCJleUpFWVhSaElqb2laRmRLY0dGWGVESldWMUpLVkVSQ2FHTkVXbHBaTUdoUVZWaENSRlZGV205a01rWklWMFpDZGxKcVJUMGlMQ0pOWldGemRYSmxiV1Z1ZENJNmV5SlFiR0YwWm05eWJTSTZJbkJzWVdsdUlpd2lRMjlrWlNJNkluQnNZV2x1SW4xOSIsImV5SkVZWFJoSWpvaVUydGFkVTFFUWt0bFZuQlJUbms1YzAxNU9XaGphbWd4V2xWU1lWTlVNR2xtVVQwOUlpd2lUV1ZoYzNWeVpXMWxiblFpT25zaVVHeGhkR1p2Y20waU9pSndiR0ZwYmlJc0lrTnZaR1VpT2lKd2JHRnBiaUo5ZlE9PSIsImV5SkVZWFJoSWpvaU1HNW9lbW8xVG5rMlZFaHFaRE5rUjBKT2JVaDVXRXh5U2toWGJFTkRWM2hIWldNM1MwVnlTSHB5YnowaUxDSk5aV0Z6ZFhKbGJXVnVkQ0k2ZXlKUWJHRjBabTl5YlNJNkluQnNZV2x1SWl3aVEyOWtaU0k2SW5Cc1lXbHVJbjE5Il19",
```

Comment:
Random comment, but i am a big fan of this new rep.

Reply:
Me too!